src/Libraries/PythonNodeModels/PythonEngineSelector .cs

```diff
@@ -20,7 +20,7 @@ public enum PythonEngineVersion
     internal sealed class PythonEngineSelector
     {
         /// <summary>
-        /// Use Lazy<![CDATA[PythonEngineSelector ]]> to make sure the Singleton class is only initialized once
+        /// Use Lazy<<![CDATA[PythonEngineSelector ]]>> to make sure the Singleton class is only initialized once
```

Comment:
I think this way it's malformed XML, so it could create problems. But I notice now Visual Studio does not handle CDATA, so we should encode the characters individually like `Lazy&lt;PythonEngineSelector&gt;`. That one worked fine for me, here is a screenshot:
<img width="1316" alt="Screen Shot 2020-07-09 at 5 50 22 PM" src="https://user-images.githubusercontent.com/10048120/87094474-f2f8a900-c20c-11ea-9794-be373a6391ce.png">

Reply:
ah @mmisol Thanks, will use that instead